src/targets/github.ts

```diff
@@ -436,31 +367,12 @@ export class GithubTarget extends BaseTarget {
     if (config.changelogPolicy !== ChangelogPolicy.None) {
       changelog = await this.getChangelog(version);
     }
-    const release = await this.getOrCreateRelease(version, revision, changelog);
 
-    const assets = await this.getAssetsForRelease(release.id);
+    const draft = await this.createReleaseDraft(version, revision, changelog);
```

Comment:
```suggestion
    const draftRelease = await this.createReleaseDraft(version, revision, changelog);
```

Reply:
Done in 7a968f67abeba6dadb462224bba5b1ed4f6fa1dc. 👍